app/models/artist.rb

```diff
@@ -2,6 +2,7 @@ class Artist < ActiveRecord::Base
   extend FriendlyId
   friendly_id :name, :use => :slugged
 
+  scope :vj, -> {where(vj: true)}
```

Comment:
adds a scope in artist, which allows us to use `Artist.vj`(vs `Artist.where(vj: true)`) – not used here